ultrasonic/src/main/kotlin/org/moire/ultrasonic/fragment/SelectAlbumFragment.kt

```diff
@@ -735,6 +763,11 @@ class SelectAlbumFragment : Fragment() {
     private fun getAlbumList(albumListType: String, albumListTitle: Int, size: Int, offset: Int) {
 
         showHeader = false
+        showSelectFolderHeader = !isOffline(context) &&
```

Comment:
If you check the SelectArtistFragment, there should be another condition to show the header: `!Util.getShouldUseId3Tags`
I think this is because if this setting is turned on, then another subsonic API will be used, which doesn't handle folders.

Reply:
It turns out that the other API, `getAlbumList2(...)`, actually does support it, but the argument was hard coded to `null`. I passed through the music folder id and it worked.